components/brave_shields/resources/cookie_list_opt_in/components/main-panel/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import { CloseIcon } from '../icons/close_icon'
+import { ShieldsIcon } from '../icons/shields_icon'
+import { getLocale } from '../../../../../common/locale'
+import Button from '$web-components/button'
+
+import * as S from './style'
+
+interface Props {
+  onEnable: () => void
+  onDismiss: () => void
+  onDecline: () => void
+  onAnimationComplete: () => void
+}
+
+function MainPanel (props: Props) {
+  const [enabled, setEnabled] = React.useState(false)
+
+  const onRootMounted = (elem: HTMLElement | null) => {
+    if (elem) {
+      elem.style.setProperty('--available-height', screen.availHeight + 'px')
+    }
+  }
+
+  const openLearnMore = () => {
+    chrome.tabs.create({
```

Comment:
I think we want to avoid using extension apis in webui. I'm not going to block on this, but I'd prefer a method in the webui mojo interface for `openLearnMore` that opens the correct url in a new tab.

Reply:
The learn more link has been removed.